README.rst

```diff
@@ -123,6 +123,8 @@ Towncrier has the following global options, which can be specified in the toml f
     single_file = true  # if false, filename is formatted like `title_format`.
     filename = "NEWS.rst"
     directory = "directory/of/news/fragments"
+    version = "1.2.3"  # project version if maintained separately
+    name = "arbitrary project name"
```

Comment:
Although now I think that this particular one can be ambigous against a section name. Especially in the context of the title_format option

Reply:
I was going to make myself a bit more familiar with this.  But, `directory` is already present in both places and I'm not sure it would make sense to include section/type-specific information in the title.  At least not without some separation like iterating over the sections or selecting one by name.  I'll follow up after my exploration.

Reply:
And I just realised I haven't pushed that commit 🤦🏻

Reply:
https://github.com/twisted/towncrier/blob/5c431028a3b699c74b162014e907272cbea8ac81/src/towncrier/build.py#L140-L143

So `name`, `version`, and `project_date` are hard coded and the only fields available to `title_format`.  I think this makes the short names match up better.  Does this satisfy your concern or did I misunderstand?

Reply:
🤦🏻 yes they make more sense. Why did I thought of it as a section name, don't know. Must have been the late hour.